content/partials/quickstart/publishing-app-store.md

```diff
@@ -23,12 +23,14 @@ publishing:
     # Configuration related to TestFlight (optional)
 
     # Optional boolean, defaults to false. Whether or not to submit the uploaded
-    # build to TestFlight beta review. Required for distributing to beta groups and expiring previous builds.
+    # build to TestFlight beta review. Required for distributing to beta groups.
     # Note: This action is performed during post-processing.
     submit_to_testflight: true 
 
-    # Optional boolean, defaults to false. Whether to expire previous builds uploaded
-    # to App Store Connect. It allows automatically submitting a new build for review if a previous submission exists.
+    # Optional boolean, defaults to false. Set to true to automatically expire all previous builds except 
+    # the current one when submitting a new build to beta review.
+    # This allows the automation of creating a new beta review submission if a prior one exists. 
```

Comment:
Can we simplify this further?

E.g. "Set to true to automatically expire all previous builds upon submitting a new build to beta review. Expired builds will no longer..."

Reply:
yes